docs/src/java/java.md

```diff
@@ -0,0 +1,495 @@
+# Java SDK
+
```

Comment:
I think it makes sense to add an admonition box here to indicate that this applies to only LanceDB Cloud and Enterprise? This would help bring attention to that fact in case OSS users stumble across it.

Here's how to add an [admonition box](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage) in mkdocs.